docs/src/pages/layout/breakpoints/breakpoints.md

```diff
@@ -7,11 +7,11 @@ Material-UI uses a **simplified** implementation of the original [specification]
 
 Each breakpoint matches with a *fixed* screen width:
 
-- **xs**, extra-small: 0px or larger
-- **sm**, small: 600px or larger
-- **md**, medium: 960px or larger
-- **lg**, large: 1280px or larger
-- **xl**, extra-large: 1920px or larger
+* **xs,** extra-small: 0px to 600px
+* **sm,** small: 600px to 960px
+* **md,** medium: 960px to 1280px
+* **lg,** large: 1280px to 1920px
```

Comment:
This is not 💯 accurate. The actual value is 1920px - 0.05px, as the intervals have an empty intersection. It's important. But 0.05px is too low-level information. I think that it's it important for people to understand that a breakpoint has an associated range, inclusive. It's better explained in 
https://material-ui.com/layout/hidden/#how-it-works. I'm not sure how we can explain it here.